### ISSUE TITLE
Add spirv-tools to github actions to enable CI testing

### ISSUE THREAD

**Issue by khyperia** [EmbarkStudios/rust-gpu#38]
tl;dr uncomment this and get it to work https://github.com/EmbarkStudios/rust-gpu/blob/c77224cecb2b4b157c3c8b11ccbbc05d100d62b6/.github/workflows/ci.yaml#L6-L31

In CI, we'd like to be able to run `cargo test`. However, that requires [spirv-tools](https://github.com/KhronosGroup/SPIRV-Tools) to be installed  on the system - e.g. we want to test if the modules we're generating are actually valid via `spirv-val`. This might be a good first issue _if_ you already know how github actions work, and how to get spirv-tools bundled into a package installable on the CI system.

If nobody picks this up in a week or so, I'll try to figure it out myself (and probably fail a lot in the process but hey I'll try)
